projects/qtaf/docs/sections/plugins/xray/Configuration.md

```diff
@@ -0,0 +1,1383 @@
+
+The Jira status to transition test execution issues to if all tests passed.
+If the status is `#!json null`, the issue will have the default issue status of issues created in the project.
+
+***Environment variable***
+: `XRAY_RESULTS_UPLOAD_CUSTOM_STATUS_TEST_EXECUTION_ISSUE_PASSED`
+
+***Type***
+: `string`
+
+***Default***
+: `#!json null`
+
+??? example
+    === "QTAF JSON"
+
+        ```json
+        "xray": {
+          "resultsUpload": {
+            "customStatus": {
+              "testExecutionIssue": {
+                "passed": "Done"
+              }
+            }
+          }
+        }
+        ```
+
+    === "Environment variable"
+        ```sh
+        XRAY_RESULTS_UPLOAD_CUSTOM_STATUS_TEST_EXECUTION_ISSUE_PASSED="Done"
+        ```
+
+<hr/>
+
+### `environments`
+
+QTAF can add test environments to created test execution issues.
+The following settings can be used to control the way QTAF assigns test environments.
+
+!!! info
+    For more information about test environments, please see:
+
+    - :fontawesome-solid-cloud:{ title="Xray cloud" .xray-icon } [Xray documentation](https://docs.getxray.app/display/XRAYCLOUD/Working+with+Test+Environments)
+    - :fontawesome-solid-server:{ title="Xray server" .xray-icon } [Xray documentation](https://docs.getxray.app/display/XRAY/Working+with+Test+Environments)
+
+<hr/>
+
+#### `driver`
+
+Toggles whether QTAF should add the name of the Selenium driver used to execute the tests as test environment.
+
+***Environment variable***
+: `XRAY_RESULTS_UPLOAD_ENVIRONMENTS_DRIVER`
+
+***Type***
+: `boolean`
+: Accepted strings (case-insensitive):
+
+    - `#!json "true"`
+      `#!json "1"`
+      `#!json "y"`
+
+    - `#!json "false"`
+      `#!json "0"`
+      `#!json "n"`
+
+***Default***
+: `#!json true`
+
+??? example
+    === "QTAF JSON"
+
+        ```json
+        "xray": {
+          "resultsUpload": {
+            "environments": {
+              "driver": false
+            }
+          }
+        }
+        ```
+
+    === "Environment variable"
+        ```sh
+        XRAY_RESULTS_UPLOAD_ENVIRONMENTS_DRIVER="false"
+        ```
+
+<hr/>
+
+#### `enabled`
+
+Toggles whether QTAF should include test environments during test execution issue creation.
+Setting this option to `false` will result in no test environments being added to the test execution issue.
+
+***Environment variable***
+: `XRAY_RESULTS_UPLOAD_ENVIRONMENTS_ENABLED`
+
+***Type***
+: `boolean`
+: Accepted strings (case-insensitive):
+
+    - `#!json "true"`
+      `#!json "1"`
+      `#!json "y"`
+
+    - `#!json "false"`
+      `#!json "0"`
+      `#!json "n"`
+
+***Default***
+: `#!json true`
+
+??? example
+    === "QTAF JSON"
+
+        ```json
+        "xray": {
+          "resultsUpload": {
+            "environments": {
+              "enabled": false
+            }
+          }
+        }
+        ```
+
+    === "Environment variable"
+        ```sh
+        XRAY_RESULTS_UPLOAD_ENVIRONMENTS_ENABLED="n"
```

Comment:
I think we do not need to use quotes in environment variables because everything is interpreted as a string by the OS.

Reply:
Yes, true. I like to use them to avoid any issues when whitespace is involved. If you want I can remove all unnecessary quotes.